patterns-use-cases/payment-state-machine/payment-state-machine-java/src/main/java/my/example/AppMain.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 - Restate Software, Inc., Restate GmbH
+ *
+ * This file is part of the Restate examples,
+ * which is released under the MIT license.
+ *
+ * You can find a copy of the license in the file LICENSE
+ * in the root directory of this repository or package or at
+ * https://github.com/restatedev/examples/
+ */
+package my.example;
+
+import dev.restate.sdk.http.vertx.RestateHttpEndpointBuilder;
+import my.example.accounts.Account;
+
+public class AppMain {
+
+    public static void main(String[] args) {
+        RestateHttpEndpointBuilder.builder()
+                .bind(new Account())
+                .bind(new PaymentService())
```

Comment:
Naming is not consistent. How about both have `Service` postfix or none of them have `Service` postfix?

Reply:
I did that because Account is an Object so it looked nicer to just call it account. But I can change it into AccountObject and PaymentService.

Reply:
Then how about `PaymentService` just becomes `Payment`?

Reply:
But the PaymentService doesn't represent a Payment. And there is already a Payment class. 
This class represents a PaymentProcessor. I can call it like that if that's better than the word service here.

Reply:
PaymentProcessor sounds better to me. I just want to avoid any confusion with our terms Service and Object.